File: SecondYear/adityshrma08_AdityaSharma_2226csit1014_2/Week_4/Encoding Message.cpp
#include <bits/stdc++.h> 
string encode(string &message)
{
    //   Write your code here.
    int count = 1;
    string ans="";
    for(int i=0;i<message.size();i++){
        if(message[i]!=message[i+1]){
            ans+=message[i]+to_string(count);
            count =1;
            continue;
        }
        count++;
    }
    return ans;
}

File: SecondYear/aayushker_AayushkerSingh_2226CSE(AI)1010_2/Week_4/Encoding_Message.py
from os import *
from sys import *
from collections import *
from math import *

def encode(message):
    # Write your code here.
    eme = ""
    c = 1

    for i in range(1, len(message)):
        if message[i] == message[i - 1]:
            c += 1
        else:
            eme += message[i - 1] + str(c)
            c = 1

    eme += message[-1] + str(c)
    return eme
    pass